turbo.json

```diff
@@ -10,51 +10,57 @@
         "generated/spec.schema.json"
       ]
     },
-    "generate:typescript": {
+    "@realm/bindgen#build": {
       "inputs": [
-        "src/*",
-        "spec.yml"
+        "src/*"
+      ],
+      "dependsOn": [
+        "generate:spec-schema"
+      ]
+    },
+    "@realm/binding#compile": {
```

Comment:
I wouldn't do this _right_ now, but at some point we may want to teach this turbo how to specifically build the different cmake outputs separately, so that eg you can do all of your TS building/rollup in parallel with compiling the C++ code.

Reply:
Sure. If we can find a way to do that - it would be great.